homeassistant/components/glances/__init__.py

```diff
@@ -1 +1,146 @@
-"""The glances component."""
+"""The Glances component."""
+from datetime import timedelta
+import logging
+
+from glances_api import Glances, exceptions
+
+from homeassistant.const import CONF_HOST, CONF_SCAN_INTERVAL
+from homeassistant.core import Config, HomeAssistant
+from homeassistant.exceptions import ConfigEntryNotReady
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.event import async_track_time_interval
+
+from .const import DATA_UPDATED, DEFAULT_SCAN_INTERVAL, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup(hass: HomeAssistant, config: Config) -> bool:
+    """Configure Glances using config flow only."""
+    return True
```

Comment:
I though that since we are moving glances to be its own integration why not just make it configured through the UI. If importing is recommend I can add the import function and the user will have to edit the config yaml file.

Reply:
Some users prefer config yaml. :man_shrugging: 

@fabaff what do you think?

Reply:
I think for a better user experience we should import the configuration to get rid of any breaking change.

Reply:
There will be a breaking fmchanfe for users in this PR, right? Is that ok @dshokouhi?

Reply:
I think that we should still support the YAML-based configuration. UI is nice and makes sense for elements you can discover and newbies but there are people who prefer to copy the configuration file from one system to another and don't want to deal with setting up the same stuff over and over again.

Reply:
Already added import option :grin:

Reply:
@MartinHjelmare yup I understand we can't escape that :)

thank you for the import option @engrbm87 !